packages/conf-freetype/conf-freetype.1/opam

```diff
@@ -7,17 +7,17 @@ license: "GPL-1.0-or-later"
 build: [["pkg-config" "freetype2"]]
 depends: ["conf-pkg-config" {build}]
 depexts: [
-  ["libfreetype6-dev"] {os-family = "debian"}
-  ["freetype-dev"] {os-distribution = "alpine"}
-  ["freetype-devel"] {os-distribution = "centos"}
-  ["freetype-devel"] {os-distribution = "fedora"}
-  ["freetype-devel"] {os-distribution = "rhel"}
-  ["libfreetype2-devel"] {os-distribution = "mageia"}
-  ["freetype2-devel"] {os-family = "suse"}
-  ["print/freetype2"] {os = "freebsd"}
-  ["print/freetype2"] {os = "openbsd"}
+  ["libfreetype6-dev"] {os-family = "debian" | os-family = "ubuntu"}
+  ["freetype-dev"] {os-family = "alpine"}
+  ["freetype-devel"] {os-family = "fedora" | os-family = "rhel" | os-family = "centos"}
+  ["freetype2-devel"] {os-family = "mandriva" | os-family = "openmandriva" | os-family = "mageia"}
+  ["freetype2-devel"] {os-family = "suse" | os-family = "opensuse" | os-family = "sles"}
+  ["freetype2"] {os-family = "arch" | os-family = "archlinux"}
+  ["media-libs/freetype"] {os-family = "gentoo"}
+  ["freetype2"] {os-family = "bsd" & os != "openbsd"}
```

Comment:
I’d rather avoid using `os-family = "bsd"` whenever possible as it is easy to introduce bugs this way. Having one line per OS makes it also easier to maintain